infrastructure/kctf-samples/apache-php/challenge/config/nsjail-php.cfg

```diff
@@ -28,6 +28,17 @@ mount: [
     dst: "/tmp"
     fstype: "tmpfs"
     rw: true
+  },
+  {
+    dst: "/mnt"
```

Comment:
this is necessary, as the /chroot is readonly

Reply:
I don't understand, can't you add RUN mkdir -p /chroot/mnt/disks/gcs-task to the Dockerfile?

Reply:
yea that would work too :P

Reply:
this is now done in Dockerfile
```
RUN chroot /chroot mkdir -p /mnt/disks/sessions /mnt/disks/uploads
```